ChangeLog

```diff
@@ -1,3 +1,14 @@
+2015-07-28  Jacob Fenton  <bocajnotnef@gmail.com>
+
+   * khmer/kfile.py: added infastructure for doing compressed output
```

Comment:
Ah, `codespell` only catches common misspellings. People should still use aspell or other checker.